### PR TITLE
Current Score and toggling between Attacker and Defender views

### PR DESCRIPTION
**Pull request by Chrisnguyen3**
Defender is just last index of the graph for now

### CODE REVIEW

File: marlon/simulate.py [hunk 1]
```diff
@@ -50,10 +51,11 @@ def run_simulation(iteration_count, agent_file):
 
         # If there is a jump in the reward for this step, record it for UI display.
         if done:
-            simulation.append(generate_graph_json(gym_env, iteration_count))
+            simulation.append(generate_graph_json(gym_env, iteration_count, current_score))
             break
         if reward != 0 or iteration == iteration_count-1:
-            simulation.append(generate_graph_json(gym_env, iteration+1))  
+            current_score += reward
+            simulation.append(generate_graph_json(gym_env, iteration+1, current_score))
```
Comment (James-LG) [inline marlon/simulate.py]: If it's possible for done to be true and for a reward to be given at the same time the last reward will be missing. Move the `current_score += reward` before the ifs?

**Comment by Chrisnguyen3** [comment]
unknown
unknown

File: web.py [hunk 2]
```diff
@@ -11,8 +11,8 @@
 @app.route('/')
 def home():
     if simulation is None:
-        return render_template('index.html', max_sim=0, num_graphs=0)
-    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation))
+        return render_template('index.html', max_sim=0, num_graphs=0, last_index=0)
+    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation), last_index=len(simulation))
```
Comment (James-LG) [inline web.py]: Can you just use `num_graphs` to determine the last index?

File: web.py [hunk 3]
```diff
@@ -26,9 +26,13 @@ def upload_file():
...
     simulation = run_simulation(ITERATION_COUNT, afile.filename)
-    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation))
\ No newline at end of file
+    print("SIMULATING {} and length of {}".format(afile.filename, len(simulation)))
+    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation), last_index=len(simulation))
+
+@app.route('/defenderView')
+def defender_view():
+    print('Defender View')
+    return render_template('index.html', max_sim=0, num_graphs=0)
```
Comment (James-LG) [inline web.py]: Is this used?

**Review by James-LG — CHANGES_REQUESTED** [review]

File: web.py (hunk 3 above)
Comment (Chrisnguyen3) [reply web.py]: Nope. Forgot to remove

File: web.py (hunk 2 above)
Comment (Chrisnguyen3) [reply web.py]: Yes... Don't know what I was think... It legit the same values next to each other...

**Review by James-LG — APPROVED** [review]

**Pull request merged**